packages/icons/src/create-pr.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 32 33" fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.786 6.449c-.513 0-.929.428-.929.957v17.223c0 .529.416.957.929.957h20.428c.513 0 .929-.428.929-.957V12.19c0-.528.416-.957.928-.957.513 0 .929.429.929.957v12.44c0 1.585-1.247 2.87-2.786 2.87H5.786C4.247 27.5 3 26.215 3 24.63V7.405c0-1.586 1.247-2.87 2.786-2.87h16.25c.512 0 .928.428.928.956s-.415.957-.928.957H5.786zm3.882 5.192a.911.911 0 011.293-.235l4.371 3.153c.66.476.66 1.484 0 1.96l-4.37 3.153a.911.911 0 01-1.294-.235.975.975 0 01.228-1.333l3.556-2.565-3.556-2.565a.975.975 0 01-.228-1.333zm8.189 6.29c-.513 0-.929.429-.929.957s.416.957.929.957H22.5c.512 0 .928-.428.928-.957 0-.528-.416-.957-.928-.957h-4.643z"/></svg>
\ No newline at end of file
+<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M6.177 3.073L8.573.677A.25.25 0 019 .854v4.792a.25.25 0 01-.427.177L6.177 3.427a.25.25 0 010-.354zM2.25 2.5a.75.75 0 100 1.5.75.75 0 000-1.5zM0 3.25a2.25 2.25 0 113 2.122v5.256a2.251 2.251 0 11-1.5 0V5.372A2.25 2.25 0 010 3.25zm10-.75H9V4h1a1 1 0 011 1v5.628a2.251 2.251 0 101.5 0V5A2.5 2.5 0 0010 2.5zM1.5 12.75a.75.75 0 111.5 0 .75.75 0 01-1.5 0zm9.5 0a.75.75 0 111.5 0 .75.75 0 01-1.5 0z" fill="currentColor"/></svg>
```

Comment:
viewBox is missing

Reply:
icon given by design team does not have viewbox